.idea/jsLinters/eslint.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="EslintConfiguration">
+    <option name="fix-on-save" value="true" />
+  </component>
+</project>
```

Comment:
This file does not match the PR title and description as it's not related to the copyright autocompletion.

Reply:
better?

Reply:
I removed the file